documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,574 @@
+  information about an aspect of the proposal.
+- "Objection": An explained uncertainty about the appropriateness, efficacy, or
+  safety of a solution
+- "Paramount objection": An objection outlining an issue that would cause harm
+  to the project or its contributors; especially issues that could cause harm
+  and are difficult to reverse and not strictly necessary to accomplish the
+  goals of the project
+- "Approved proposal": A proposal that has undergone the formal decision-making
+  process and has been accepted as a valid solution by the participants of the
+  discussion.
+- "Process proposal": A proposal to change processes followed by the
+  contributors in the course of stewarding the project towards success.
+- "Purpose": The motivation for the proposal. This is typically a statement
+  describing the problem the proposal aims to solve. In the original Sociocratic
+  model this is called the "driver".
+
+## Process summary
+
+The process breaks proposal evaluation and decision-making into a series of
+rounds. Each round has expectations about what kinds of feedback are
+appropriate. These steps are ordered and must be followed in the order they are
+presented here.
+
+Full descriptions for each of these rounds are below. This section is a summary
+meant to give an overview for people new to or refreshing themselves on the
+process.
+
+> **Note** The "goal" column is a very brief summary and is not intended to
+> encapsulate the full scope of the round. Please refer to the
+> [round descriptions below](#round-descriptions) for more details and examples.
+
+| round                                                 | Suggested span      | Goal                                                                                      |
+| ----------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| [Question round](#question-round)                     | 3 days              | Clarify the proposal; share initial reactions                                             |
+| [Revision round](#revision-round)                     | Author's discretion | Update the text of the proposal to reflect the outcome of the previous round              |
+| [Objection round](#objection-round)                   | 2 days              | Identify paramount and non-paramount objections                                           |
+| [Objection revision round](#objection-revision-round) | Author's discretion | Work with participants to revise the text of the proposal to address paramount objections |
+| [Approval](#approval)                                 | N/A                 | Mark as approved and create issues to implement the proposal                              |
+| [Tabling](#tabling)                                   | N/A                 | Indicate that a proposal will not be implemented                                          |
```

Comment:
Yup! I didn't include it because I wasn't sure if MyST supported mermaid diagrams but it does (https://myst-tools.org/docs/mystjs/diagrams) so I'll add it during revision.